### ISSUE TITLE
Can't run TensorBoard on El Captain

### ISSUE THREAD

**Issue by VikramTiwari** [tensorflow/tensorflow#35]
Got this as an issue while trying to run TensorBoard on El Captain.

```
python tensorflow/tensorboard/tensorboard.py
/usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: can't open file 'tensorflow/tensorboard/tensorboard.py': [Errno 2] No such file or directory
```

Even after following instructions [here] (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#on-macosx-) for installing tensorflow on a brew based installation of python the error remains same:

```
python tensorflow/tensorboard/tensorboard.py
python: can't open file 'tensorflow/tensorboard/tensorboard.py': [Errno 2] No such file or directory
```

**Comment by danmane** [comment]
A few questions:
* How did you install TensorFlow?
* What directory are you running that command in?
* What happens if you open a python interpreter and type `import tensorflow as tf`?

**Comment by manning** [comment]
Isn't this just that if you did the binary install, you don't have a `tensorflow/tensorboard/tensorboard.py` file on that relative path, and you should instead be using the `tensorboard` command?

**Comment by VikramTiwari** [comment]
To answer @danmane 
1. I used the process of installing on OSX using HomeBrew installation of Python and then TensorFlow using `pip`
2. The directory is very generic but I have tried running it on multiple directories which were owned by user.
3. Running the sample python program on the getting started page works without any issues.

@manning While I was writing this, I did a fresh installation to it and it works with
`tensorboard --logfile ./` rather than `tensorflow/tensorboard/tensorboard.py --logging ./` as I was trying earlier.

I guess that was the only issue. But I will try to replicate it later and provide anymore details.
